app/javascript/components/server-components/LibraryPage.tsx

```diff
@@ -344,6 +344,30 @@ const LibraryPage = ({ results, creators, bundles, reviews_page_enabled, followi
             )}
           </div>
         ) : null}
+        {hasArchivedProducts && !state.search.showArchivedOnly ? (
+          <div
+            className="archived-banner"
+            style={{
+              marginBottom: "20px",
+              padding: "12px",
+              backgroundColor: "#f8f9fa",
+              border: "1px solid #dee2e6",
+              borderRadius: "4px",
+              textAlign: "center",
+            }}
+          >
+            You have archived purchases-{" "}
```

Comment:
list out the number and add e2e spec for this new behavior to library request specs